src/rdkafka_mock.c

```diff
@@ -920,6 +928,23 @@ static void rd_kafka_mock_cluster_io_add(rd_kafka_mock_cluster_t *mcluster,
         mcluster->fd_cnt++;
 }
 
+/**
+ * @brief Reassign partition replicas to broker, after deleting or
+ *        adding a new one.
+ */
+static void
+rd_kafka_mock_cluster_reassing_partitions(rd_kafka_mock_cluster_t *mcluster) {
```

Comment:
```suggestion
rd_kafka_mock_cluster_reassigning_partitions(rd_kafka_mock_cluster_t *mcluster) {
```

Reply:
Maybe `rd_kafka_mock_cluster_reassign_partitions` with the typo fixed? `reassigning_partitions` seems more a question than a command

Reply:
Works.